handbook/openid-connect/openid-connect.md

```diff
@@ -188,6 +209,14 @@ SystemLink supports the following algorithms for ID token signing, ID token key
 
 - AES-256 Key Wrap
 
+Algorithms which require a private key:
+
+- RSAES OAEP
+
+- ECDH-ES
+
+Refer to [**SystemLink Login Configuration**](#systemlink-login-configuration) for information on configuring the private key.
```

Comment:
I assume this link works better when running via mkdocs? Following that link in the github md previewer produces some weird behavior. Same for other links.

Reply:
For me they are working both in mkdocs and when you're looking at the rendered file in the [repo](https://github.com/ni/systemlink-operations-handbook/blob/configure-asymmetric-id-token-encryption-key/handbook/openid-connect/openid-connect.md). It didn't work in the "rich diff" view in the PR.